packages/turf-transform-translate/index.js

```diff
@@ -0,0 +1,174 @@
+var helpers = require('@turf/helpers');
+var point = helpers.point;
+var multiPoint = helpers.multiPoint;
+var lineString = helpers.lineString;
+var multiLineString = helpers.multiLineString;
+var polygon = helpers.polygon;
+var multiPolygon = helpers.multiPolygon;
+var feature = helpers.feature;
+var invariant = require('@turf/invariant');
+var getCoords = invariant.getCoords;
+var rhumbDestination = require('@turf/rhumb-destination');
+// var dist = require('@turf/distance');
+// var rhumbDistance = require('@turf/rhumb-distance');
+
+/**
+ * Moves any geojson Feature or Geometry of a specified distance on the provided direction angle.
+ *
+ * @name translate
+ * @param {Geometry|Feature<any>} geojson object to be translated
+ * @param {number} distance length of the motion; negative values determine motion in opposite direction
+ * @param {number} direction of the motion; angle from North between -180 and 180 decimal degrees, positive clockwise
+ * @param {string} [units=kilometers] in which `distance` will be express; miles, kilometers, degrees, or radians
+ *     (optional, default `kilometers`)
```

Comment:
`(optional, default "kilometers")` is not required since it's already defined as optional using the square brackets.

```javascript
/**
 * @param {string} [units=kilometers]
```